test/b2gedmntuples_cfg.py

```diff
@@ -768,7 +768,7 @@
 process.boostedJetUserDataAK8 = cms.EDProducer(
     'BoostedJetToolboxUserData',
     jetLabel  = cms.InputTag('jetUserDataAK8'),
-    puppiSDjetLabel = cms.InputTag('packedPatJetsAK8PFPuppiSoftDrop'),
+    puppiSDjetLabel = cms.InputTag('slimmedJetsAK8'),
```

Comment:
@jkarancs are the above for corrected or uncorrected subjets? 
It seems 4% of jets have subjets missing when reclustered?

Small differences are expected if jets are reclustered from MiniAOD. While it is true that the corrections are based on the original MiniAOD, I think we should use only one jet collection in the framework to be consistent.